section-09-differential-testing/research-notebook/08-final-machine-learning-pipeline.ipynb

```diff
@@ -0,0 +1,1124 @@
+{
```

Comment:
@solegalli please check you're happy with the notebook - it's almost identical, just a few additional lines to output the prediction csv file

Reply:
Looks good, thank you!